src/pages/create-quiz-page/create-quiz.tsx

```diff
@@ -1,7 +1,180 @@
-import React from "react";
+import React, { useState } from "react";
+import { Button } from "primereact/button";
+import { RadioButton } from "primereact/radiobutton";
+import { InputText } from "primereact/inputtext";
+import { Dropdown } from "primereact/dropdown";
+import { Card } from "primereact/card";
+import "./create-quiz.scss";
+import { Header } from "../../common/components/header/header";
 
 interface CreateQuizProps {}
 
 export const CreateQuiz: React.FC<CreateQuizProps> = ({}) => {
-  return <div></div>;
+  const [answer, setAnswer] = useState(null);
+
+  const HeaderCard1 = (
+    <div className="flex flex-row align-items-center justify-content-between">
+      <div className="create-quiz__title-wrapper">
```

Comment:
The title and subtitle are elements which are being used in multiple places in the application. You could extract them into separate common components and use them wherever needed.